charts/newrelic-logging/values.yaml

```diff
@@ -169,7 +169,7 @@ rbac:
 
 serviceAccount:
   # Specifies whether a ServiceAccount should be created
-  create: true
+  create:
```

Comment:
Would it make sense to include a comment here that in order for this to be `boolean false` it needs to be empty, due to [this](https://github.com/newrelic/helm-charts/blob/master/library/common-library/templates/_serviceaccount.tpl#L11-L13)?

Also, the mention to `include "newrelic-logging.serviceAccount"` from [here](https://github.com/newrelic/helm-charts/blob/master/library/common-library/templates/_serviceaccount.tpl#L12) is obsolete now, please replace it by `include "newrelic.common.serviceAccount.name"`

Reply:
> Would it make sense to include a comment here that in order for this to be boolean false it needs to be empty, due to [this](https://github.com/newrelic/helm-charts/blob/master/library/common-library/templates/_serviceaccount.tpl#L11-L13)?

We can comment it. That it has to be `nil` so the common library manages the possibility to set it globally. But in the end, it defaults to true [here](https://github.com/newrelic/helm-charts/blob/ebf870a/library/common-library/templates/_serviceaccount.tpl#L34-L37
)

> Also, the mention to include "newrelic-logging.serviceAccount" from [here](https://github.com/newrelic/helm-charts/blob/master/library/common-library/templates/_serviceaccount.tpl#L12) is obsolete now, please replace it by include "newrelic.common.serviceAccount.name"

This is a comment that was copied and pasted ages ago and it managed to enter the common library because the PRs were huge. But in the end, it makes no difference at all.

The tests for charts do not parse the files, simply look for changes in files so making that change means having to re-release the common library and upgrade all the charts. It can wait for the next issue/feature to do it.

Reply:
Yeah, regarding the second comment, I'm totally OK with this: let's wait for the next release to introduce this change.